app/core/modals/SupportingMemberSignupModal.tsx

```diff
@@ -12,7 +12,7 @@ function classNames(...classes) {
   return classes.filter(Boolean).join(" ")
 }
 
-export default function SignupModal({ button }) {
+export default function SignupModal({ button, sponsored = false }) {
```

Comment:
The name of the component here is `SignupModal`, but the filename here is `SupportingMemberSignupModal`. Is this intended? (I'm assuming that it's better to keep these two the same.)

Reply:
That should be updated indeed, bad habit of mine.